tasks/21_check-letters/check-letters.js

```diff
@@ -0,0 +1,6 @@
+export function checkLetters(arr) {
+  let includedLetters = arr[1].toLowerCase().split('')
+      .map((a) => arr[0].toLowerCase().indexOf(a))
+      .filter((a) => a !== -1);
+  return includedLetters.length === arr[1].length;
+}
```

Comment:
please give meaningful variables for `arr[0]` and `arr[1]`. extract `.toLowerCase()` from iteration. Use `.every` or `.some`. and probably it's better to use `.include` instead of `.indexOf` despite of the link in description